app/components/QuestionForm/index.js

```diff
@@ -121,11 +115,19 @@ export const QuestionForm = ({
   communityQuestionsType,
   disableCommForm,
 }) => {
+ const [isSelectedType, setIsSelectedType] = useState(false);
+ const [isError, setIsError] = useState(false);
+
   const handleSubmitWithType = sendQuestion => {
     if (communityQuestionsType !== ANY_TYPE) {
       change(FORM_TYPE, communityQuestionsType);
+    } 
+    if(!isSelectedType && !isError){
+      setIsError(true);
     }
-    return handleSubmit(sendQuestion);
+    if(!isError && isSelectedType){
+        return handleSubmit(sendQuestion);
```

Comment:
and here just return `handleSubmit(sendQuestion);`
without if block

Reply:
isError is nessesery